toxav/toxav.h

```diff
@@ -246,9 +246,9 @@ typedef enum Toxav_Err_Call {
  * receiving are both enabled by default.
  *
  * @param friend_number The friend number of the friend that should be called.
- * @param audio_bit_rate Audio bit rate in Kb/sec. Set this to 0 to disable
+ * @param audio_bit_rate Audio bit rate in kbit/sec. Set this to 0 to disable
```

Comment:
Kilobit is written as "kb" in SI units. Second is written as "s" in SI units. You have improved one part, i.e. lower-cased the "k", but "bit" isn't anything ("t" might be tonnes, "T" might be tesla). What's the intention here?

Reply:
My brain needs the "bit" spelled out, or it will read "byte". I can make it kilobit if you find that more acceptable.

Reply:
also this commend kinda reminds me of what nurupo said above.